src/components/MyAccount/RequestsTab/FreezeButton.tsx

```diff
@@ -40,6 +43,15 @@ const FreezeButton = ({ hold, patron }: { hold: Hold; patron: Patron }) => {
   const successModalProps = {
     type: "default",
     closeButtonLabel: "OK",
+    bodyContent: (
+      <Box className={styles.modalBody}>
+        <Text>
+          Your hold on this item has been frozen. You will continue to advance
```

Comment:
This works but it says "has been frozen" for both the "Freeze" and "Unfreeze" actions. So there should be a conditional for the word.

Reply:
good catch.